ws/handler.go

```diff
@@ -66,7 +86,7 @@ func WebsocketHandler(headersToFetch []string, subprotocols []string, config *Co
 			url = fmt.Sprintf("%s%s%s", scheme, r.Host, url)
 		}
 
-		info, err := NewRequestInfo(r, headersToFetch)
+		info, err := NewRequestInfo(r, headersExtractor)
```

Comment:
I think, we should pass `headersExtractor` here by reference (`&headersExtractor`) to avoid copying it for every request.

Reply:
@palkan Done!